tests/test_cli/test_upgrade.py

```diff
@@ -1185,7 +1185,12 @@ def test_upgrade(self, mock_confirm, confirm):
             self.current_agent_context, self.EXPECTED, ignore=ignore
         )
         _left_only, _right_only, diff = dircmp_recursive(dircmp)
-        assert diff == _left_only == _right_only == set()
+        if confirm:
+            assert diff == _right_only == _left_only == set()
+        else:
+            assert diff == _right_only == set()
+            # temporary: due to change in deps
+            assert _left_only == {"vendor/fetchai/skills/error"}
```

Comment:
@marcofavorito  a hack, not sure you have a better idea.

Reply:
yeah it's OK